### DISCUSSION TITLE
챌린지 4,5번(웹 애플리케이션 주소, 저장소) 제출 양식 질문드립니다.

### DISCUSSION THREAD

**Discussion by solidcellaMoon** [devrel-kr/HackaLearn#227] [1 Q&A - HackaLearn 운영] [answered]
안녕하세요! 챌린지 4, 5번(정적 웹앱 프로젝트) 제출 양식에 대해 질문드립니다. 프로젝트 완성 후에 모든 팀원이 각각 완성물 링크를 PR해야 하는건지, 팀 대표 한명이 제출해도 전원 제출로 인정되는건지 궁금합니다.

또한 아래 첨부한 챌린지 5번 안내문 내용에서, 설치 및 배포 방법의 의미에 대해 질문 드립니다. 리뷰어가 직접 Azure Portal과 Azure Static Web Apps를 이용해 배포하는 과정을 설명해야 하는지 궁금합니다. 아니면 저장소 코드를 받아와 로컬에서 실행하는 과정을 작성해도 되는 걸까요? 

> NOTE: 리포지토리의 README.md 파일은 반드시 **설치 및 배포 방법**에 대한 설명이 포함되어 있어야 합니다. 리뷰어가 설명대로 직접 실행해서 확인합니다.

**Comment by justinyoo** [comment, marked as answer]
@solidcellaMoon .

1. 먼저 첫번째 질문에 답을 드리겠습니다. 챌린지 4번과 5번은 팀당 한 분만 제출하셔도 전체 팀에 적용이 됩니다. 팀으로 작업한 애저 정적 웹 앱 URL이고, 리포지토리이니까요.
2. 5번 챌린지를 보면 README.md 파일에 설치 방법과 배포 방법에 대한 안내를 요구했습니다. 할당된 리뷰어가 해당 리포지토리를 로컬로 클론한 후 README.md 파일에 적혀 있는 설명대로 환경설정을 하면 로컬에서 동작해야 합니다. 마찬가지로, 해당 리포지토리를 클론 받아서 리뷰어가 애저로 배포할 경우에도 README.md 파일의 설명대로 설정하면 문제 없이 배포가 가능해야 합니다. 이는 협업의 과정에서 항상 발생하는 "나는 되는데 너는 안되네?"의 문제를 해결하기 위한 것이라고 보시면 됩니다.

**Reply by justinyoo** [reply]
만약 답이 되었다면 우측 상단의 체크박스를 클릭해 주세욥! 😉
